tests/Integration/Service/MailTransmissionIntegrationTest.php

```diff
@@ -159,6 +163,22 @@ public function testSendMailWithCloudAttachment() {
 		$this->addToAssertionCount(1);
 	}
 
+	/* TODO
+	public function testSendMailWithAttachmentFromAttachment() {
```

Comment:
dead? why is this commented out?

Reply:
didn't implement it yet (don't know if I will though)

Reply:
then just drop that code and we add the test when the actual feature is added :)